docs/adding_a_dataset.md

```diff
@@ -0,0 +1,252 @@
+|                         | Political                | Categorizing text according to political orientation or content.                                                |
+|                         | Language Identification  | Determining the language in which a given piece of text is written.                                             |
+|                         | Linguistic Acceptability | Assessing whether a sentence is grammatically correct according to linguistic norms.                            |
+|                         | Sentiment/Hate Speech    | Detecting the sentiment of text or identifying hate speech within the content.                                  |
+|                         | Dialog Systems           | Creating or evaluating systems capable of conversing with humans in a natural manner.                           |
+| **Clustering**          |                          | Clustering involves grouping sets of texts together based on their similarity without pre-defined labels.       |
+|                         | Thematic Clustering      | Grouping texts based on their thematic similarity without prior labeling.                                       |
+| **Reranking**           |                          | Reranking adjusts the order of items in a list to improve relevance or accuracy according to specific criteria. |
+| **Pair Classification** |                          | Pair classification assesses relationships between pairs of items, such as texts, to classify their connection. |
+| **STS**                 |                          | Semantic Textual Similarity measures the degree of semantic equivalence between two pieces of text.             |
+
+
+</details>
+
+
+
+## submit a PR
+
+Once you are finished create a PR to the [MTEB](https://github.com/embeddings-benchmark/mteb) repository. If you haven't created a PR before please refer to the [GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/)
+
+The PR will be reviewed by one of the organizers or contributors who might ask you to change things. Once the PR is approved the dataset will be added into the main repository. 
+
+
+Before you commit here is a checklist you should consider completing before submitting:
+
+- [ ] I have tested that the dataset runs with the `mteb` package.
+
+An easy way to test it is using:
+```python
+from mteb import MTEB
+from sentence_transformers import SentenceTransformer
+
+# Define the sentence-transformers model name
+model_name = "sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2"
+
+model = SentenceTransformer(model_name)
+evaluation = MTEB(tasks=["{the name of your task}"])
+```
+
+- [ ] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
+  - [ ] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
+  - [ ] `intfloat/multilingual-e5-small`
```

Comment:
Do we want to add a sentence re: non-triviality and non-impossibility? I.e. we do not accept datasets where there is only a trivial performance difference between these two models.

Reply:
This is extremely hard to formalise, though, but makes the point system much harder to game.

Reply:
I would leave it for the review.

Reply:
Maybe PR template when we launch?

Reply:
Added